src/main/java/com/github/manolo8/darkbot/backpage/GalaxyManager.java

```diff
@@ -0,0 +1,92 @@
+package com.github.manolo8.darkbot.backpage;
+
+import com.github.manolo8.darkbot.Main;
+import com.github.manolo8.darkbot.backpage.entities.galaxy.*;
+import com.github.manolo8.darkbot.utils.XmlHelper;
+import org.dom4j.Document;
+import org.dom4j.Element;
+import org.dom4j.io.SAXReader;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GalaxyManager {
+
+    public GalaxyInfo galaxyInfo;
+    private BackpageManager backpageManager;
+    private Main main;
+
+    public GalaxyManager(Main main, BackpageManager backpageManager) {
+        this.main = main;
+        this.backpageManager = backpageManager;
+        this.galaxyInfo = new GalaxyInfo();
+    }
+
+    public void performGateSpin(String gateName, boolean sample, int gateId, int multiplier, int minWait) {
+        String params = "flashinput/galaxyGates.php?userID=" + main.hero.id + "&action=multiEnergy&sid=" + main.statsManager.sid + "&gateID=" + gateId + "&" + gateName + "=1";
+        if (sample) params = params + "&sample=1";
+        if (multiplier > 0) params = params + "&multiplier=" + multiplier;
+
+        parseJumpgate(params, minWait);
+    }
+
+    public void updateJumpgateInfo(int minWait) {
+        parseJumpgate("flashinput/galaxyGates.php?userID=" + main.hero.id + "&action=init&sid=" + main.statsManager.sid, minWait);
+    }
+
+    private void parseJumpgate(String params, int minWait) {
```

Comment:
This whole method is pretty inconsistent.
You define variables at the top, but set them only inside their if-blocks, but for some reason energy cost is ouside of one

make this follow some sensible standard.

Probably should make a method to update each one of them, and just have 4 method calls in this one, passing the elementIterator as parameter.